seeds/data/json/teamtype.json

```diff
@@ -0,0 +1,66 @@
+[
+  {
+    "name": "iOS App Team",
+    "description": "You are developing a new line of mobile applications for the Rainforest system. These applications will be on iOS. They interact primarily with the Rainforest API, but also cache important data such as payment data and various other personal information. These applications involve a combination of both native implementations and HTML5 implementations on mobile devices.\nYour products will be considered mature at the beginning of Month 3.\nYour products are considered to be public-facing, meaning untrusted users outside the company will interact with your product.\nYour product primarily interacts with API, with some interaction with Cloud and Web.\nGoal: Devote at least 30 devcaps of Routine Development by the end of Month 3, or this team will be fired.",
+    "initial_mature": false,
+    "initial_resources": 10,
+    "initial_mindset": 0,
+    "disabled": false
+  },
+  {
+    "name": "Android Open Source App Team",
+    "description": "You are developing a new line of mobile applications for the Rainforest system. These applications will be on Android devices. They interact primarily with the Rainforest API, but also cache important data such as payment data and various other personal information. These applications involve a combination of both native implementations and HTML5 implementations on mobile devices.\nUnlike the iOS team, this product is unique because it’s also open source. This means that you can leverage a community of volunteers to inspect your code.\nSpecifically, you are able to get code inspections of new development changes each month (#3), with no devcap cost (as a results you get the Hacker Mindset bonus too). Track this on your sheet by changing the Dev Capacity entry the Tasks page to zero.\nYour products will be considered mature at the beginning of Month 3.\nYour products are considered to be public-facing, meaning untrusted users outside the company interact with your product.\nYour product primarily interacts with API, with some interaction with Infrastructure and Web.\nGoal: devote at least 30 devcaps of Routine Development by the end of Month 5, or this team will be fired.",
```

Comment:
No clue as to why GitHub highlighting isn't working on some of these description fields. jsonvalidator says it's valid. My guess is the string length.